sitemap.xml

```diff
@@ -3,7 +3,7 @@ layout: null
 ---
 <?xml version="1.0" encoding="UTF-8"?>
 <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
-  {%- assign stable_pages_names = site.pages | where_exp: "stable_pages", "stable_pages.url contains site.versions['stable']" | map: "name" -%}
+  {%- assign stable_pages_names = site.pages | where_exp: "stable_pages", "stable_pages.url contains site.versions['stable']" | where_exp: "stable_pages", "stable_pages.name != 'index.md'" | map: "name" -%}
```

Comment:
This is here because we want to include /docs/dev/index.md in the sitemap so that any new dev pages can be reached by Google and other search engines. The logic I built here basically says exclude all stable page names that are in the list, but we're excluding something from an exclude list, which means it's being included instead.